src/index.js

```diff
@@ -1,30 +1,8 @@
-import { Component } from 'react';
 import { render } from 'react-dom';
-import { Provider } from 'react-redux'
-
 import store from 'redux/store';
-
-import TodoHeader from 'components/todo-header';
-import TodoList from 'components/todo-list';
-import AddTodo from 'components/add-todo';
-import TodoFilter from 'components/todo-filter';
-
-class TodoApp extends React.Component {
-  render() {
-    return(
-      <div>
-        <TodoHeader />
-        <AddTodo />
-        <TodoList />
-        <TodoFilter />
-      </div>
-    );
-  }
-};
+import Root from 'components/root';
```

Comment:
What is the point of having both `Root` and `index.js` I think you can put everything you have in the `Root` component here.

Reply:
There is not really much other benefit except keeping the entry point`index.js` very small.

Reply:
I'll research tomorrow what are the best practices for this component.